tools/taskcluster/get-bitbar-token.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script talks to the taskcluster secrets service to obtain the
+Bitbar token and write it to the .bitbar_token file in the root
+directory.
+"""
+
+import os
+import taskcluster
+import json
+
+# Get JSON data from taskcluster secrets service
+secrets = taskcluster.Secrets({'baseUrl': 'http://taskcluster/secrets/v1'})
```

Comment:
What will happen if this network request fails?  Will it cancel the build?

Reply:
No idea actually. How is this handled in other projects? @npark-mozilla ?

Reply:
The build will fail, and that's good, we want to know when this service fails, since we rely on it.  We do the same for the pocket/sentry key, accessing the same URL.  I think this is good.

Reply:
Agreed, I just wanted to make sure it wouldn't silently fail.